addons/ShaderFunction-Extras/Utility/utility.gdshaderinc

```diff
@@ -51,3 +92,11 @@ float fresnel(float amount, vec3 normal, vec3 view) {
 vec3 fresnel_glow(float amount, float intensity, vec3 color, vec3 normal, vec3 view) {
 	return pow((1.0 - dot(normalize(normal), normalize(view))), amount) * color * intensity;
 }
+
+float smooth_clamp(float x, float a, float b) {
```

Comment:
`x` should be renamed to `IN` or `value`. `a` and `b` are the min and max values I presume? They should be renamed accordingly